src/components/CustomizableAvatar.js

```diff
@@ -0,0 +1,17 @@
+import Image from 'next/image'
+function CustomizableAvatar({
+  customClass,
+  src,
+  height,
+  width,
+  alt,
+  ...props
```

Comment:
I think you might have to look at this. Not sure there's an active use for ...props?
Could you clarify?

Reply:
I changed ...props to a class name since we wouldn't have that many additional props to the avatar @annonymousauthority 
<img width="675" alt="Screenshot 2023-04-20 at 11 03 17" src="https://user-images.githubusercontent.com/72883059/233316405-1c4d93b4-8ee8-4d1b-b029-2b382bad737f.png">